05_testing_and_ci/intro_slides.md

```diff
@@ -29,9 +29,12 @@ slideOptions:
 
 ## Learning Goals of the Chapter
 
-- Justify the effort of developing testing infrastructure for simulation software
-- Discern the concepts of unit testing, integration testing and regression testing with the perspective of simulation software
-- Work with the Python testing frameworks `pytest` and `unittest`
+- Know the need of developing testing infrastructure for simulation software.
```

Comment:
```suggestion
- Explain why developing tests is crucial.
```
Infrastructure sounds too much like we would develop testing software. Any reason why should make this statement specific to simulation software? Any "know" is too weak here IMO.